.evergreen/run-tests.sh

```diff
@@ -28,6 +27,10 @@ else
   export ASYNC_TYPE="-Dorg.mongodb.async.type=nio2"
 fi
 
+# We always compile with the latest version of java
```

Comment:
Removed the redundant JAVA_HOME on line 20, but this is needed higher up so that keytool can be found (on line 46).

Reply:
If JAVA_HOME is always pointing to JAVA 8 - why do we need this? and how did it work before when it was declared after?

Reply:
`JAVA_HOME` is unset when the script starts.

What was happening before was that `JAVA_HOME` was set to `/opt/java/${JDK}` and that was used as the JAVA_HOME to locate `keytool`.  Then `JAVA_HOME` was set to `/opt/java/jdk8` and that was used to run `gradlew`.  And `/opt/java/${JDK}` was used to run to actually execute the tests.

But that's a bit more complicated than we need.  We can just use `/opt/java/jdk8` for both `keytool` and `gradlew`, and `/opt/java/${JDK}` for running the tests.

Reply:
thanks - that is complex!